pyedflib/highlevel.py

```diff
@@ -191,13 +192,23 @@ def make_header(technician='', recording_additional='', patientname='',
         del now
     if isinstance(birthdate, datetime):
         birthdate = birthdate.strftime('%d %b %Y').lower()
+
+    # backwards compatibility
+    if gender is not None:
+        if sex == '':
+            sex = gender
+        elif gender != sex:
+            raise ValueError(f'Passed both sex and gender, with different values: "{sex}" != "{gender}"')
+    del gender
+
```

Comment:
give deprecation warning is gender is set instead of sex

Reply:
Done.